cime_config/machines/config_machines.xml

```diff
@@ -3417,15 +3417,20 @@
       </modules>
 
       <modules compiler="gnu">
-        <command name="load">PrgEnv-gnu/8.3.2</command>
-        <command name="load">gcc/11.2.0</command>
+        <command name="load">PrgEnv-gnu/8.3.3</command>
+        <command name="load">gcc/12.1.0</command>
       </modules>
 
       <modules compiler="nvidia">
-        <command name="load">PrgEnv-nvidia</command>
+        <command name="load">PrgEnv-nvidia/8.3.3</command>
         <command name="load">nvidia/22.3</command>
       </modules>
 
+      <modules compiler="intel">
+        <command name="load">PrgEnv-intel/8.3.3</command>
+        <command name="load">intel-classic/2022.2.1</command>
```

Comment:
@jonbob, have you been able to build E3SM successfully with this configuration?  If so, I'd like to add it to mache (and compass and polaris).

Reply:
@xylar -- yes, and this is what I gave to Irena for her recent runs.

Reply:
Ah, okay.  I hadn't been paying attention.  I assumed she was building with `gnu` still.

Reply:
Oh, my bad -- I wasn't paying attention to the specifics of your question. No, I still haven't gotten it to run with intel, but wanted to have the correct module for testing and further work

Reply:
In her build script, she isn't specifying a compiler so I think it's defaulting to `gnu` still.

Reply:
(Unless that got changed from the version of the build script I saw last.)

Reply:
> No, I still haven't gotten it to run with intel, but wanted to have the correct module for testing and further work.

Okay I'll hold off on this until I hear how things go.